images/fcos-bastion-image/root/usr/bin/clean-up.sh

```diff
@@ -36,6 +36,7 @@ do
   fi
   echo "<3>$cluster is more than 24 hours old and not preserved, starting the pruning...."
   prune_nodes "$cluster"
+  rm -rf /opt/registry-"${port}"/data/docker/registry/v2/repositories/"$cluster"
```

Comment:
Can this be part of normal deprovisioning step by adding it to baremetal-lab-post chain and add it to `images/prow-image/root/usr/bin/deprovision.sh`? Then it does not need to be included separately in this script. It will happen on every deprovisioning and when prune_nodes is run.